modules/cassandra/cassandra.go

```diff
@@ -0,0 +1,116 @@
+package cassandra
+
+import (
+	"context"
+	"io"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/go-connections/nat"
```

Comment:
Gci shall require a new line here. 
Which IDE are you using ?

Reply:
Im using vscode but also i checked in GoLand, it did not do anything or warn about it. But I added new line manually. Thnak you

Reply:
@mdelapenya , shall an issue be created ? This is the second module where the linting is not applied.
@anilsenay , Is golangci-lint installed on your machine ?
In vscode, are you using the workspace configuration from [here](https://github.com/testcontainers/testcontainers-go/blob/main/.vscode/.testcontainers-go.code-workspace) ?
In your settings, do you have this: 
```
"go.lintTool": "golangci-lint"
```

Reply:
I wonder if we could run golangci-lint when a module is generated, but instead of using the binary (see Makefile) calling the right Docker image to avoid having to install it on every developer's machine.

Reply:
Not every machine has docker, on windows for exemple, it's not installed by default.

Reply:
Yeah, but installing docker is easier than installing a binary for linting. In any case, I'm biased from my past experience in a release engineering team, where we dockerised everything to run smoothly on the CI 😅